src/modules/Invoice/Service.php

```diff
@@ -1251,7 +1251,8 @@ public function generatePDF($hash, $identity)
 
         if (isset($company['logo_url']) && !empty($company['logo_url'])) {
             $url = parse_url($company['logo_url'], PHP_URL_PATH);
-            if (!file_exists($url)) {
+            // prevent openbase error from preventing pdf creation when debug mode is enabled
+            if (@!file_exists($url)) {
```

Comment:
I am not a fan of it. But at least it prevents dompdf not working. 

Other one is no "issue" as it is falling inside open base dir